### ISSUE TITLE
CRC setup failing to deploy correctly [BUG]

### ISSUE THREAD

**Issue by tumimo-cloud** [code-ready/crc#2837]
### General information

  * OS: Linux 
  * Hypervisor: KVM / hyperkit
  * Did you run `crc setup` before starting it (Yes)?
  * Running CRC on:  VM (vmware)

## CRC version
```bash
# Put `crc version` output here
```
CodeReady Containers version: 1.34.0+34c31851
OpenShift version: 4.9.0 (not embedded in executable)

  
## CRC status
```bash: CRC: command not found...
# Put `crc status --log-level debug` output here
```
DEBU CodeReady Containers version: 1.34.0+34c31851 
DEBU OpenShift version: 4.9.0 (not embedded in executable) 
DEBU Running 'crc status'                         
DEBU Checking file: /root/.crc/machines/crc/.crc-exist 
Machine does not exist. Use 'crc start' to create it

## CRC config
```bash
# Put `crc config view` output here
```
- consent-telemetry                     : no

## Host Operating System
```bash
# Put the output of `cat /etc/os-release` in case of Linux
# put the output of `sw_vers` in case of Mac
# Put the output of `systeminfo` in case of Windows
```
NAME="Red Hat Enterprise Linux"
VERSION="8.5 (Ootpa)"
ID="rhel"
ID_LIKE="fedora"
VERSION_ID="8.5"
PLATFORM_ID="platform:el8"
PRETTY_NAME="Red Hat Enterprise Linux 8.5 (Ootpa)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:redhat:enterprise_linux:8::baseos"
HOME_URL="https://www.redhat.com/"
DOCUMENTATION_URL="https://access.redhat.com/documentation/red_hat_enterprise_linux/8/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"

REDHAT_BUGZILLA_PRODUCT="Red Hat Enterprise Linux 8"
REDHAT_BUGZILLA_PRODUCT_VERSION=8.5
REDHAT_SUPPORT_PRODUCT="Red Hat Enterprise Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="8.5"

### Steps to reproduce

  1. 
  2. 
  3. 
  4. 

### Logs
# $ crc start --log-level debug
DEBU CodeReady Containers version: 1.34.0+34c31851 
DEBU OpenShift version: 4.9.0 (not embedded in executable) 
DEBU Running 'crc start'                          
DEBU Total memory of system is 10074533888 bytes  
/home/admin/.crc/cache/crc_libvirt_4.9.0.crcbundle not found, please provide the path to a valid bundle using the -b option

# $ crc setup --log-level debug

DEBU CodeReady Containers version: 1.34.0+34c31851 
DEBU OpenShift version: 4.9.0 (not embedded in executable) 
DEBU Running 'crc setup'                          
DEBU Checking if systemd-resolved.service is running 
DEBU Running 'systemctl status systemd-resolved.service' 
DEBU systemd-resolved.service is already running  
INFO Checking if running as non-root              
INFO Checking if running inside WSL2              
INFO Checking if crc-admin-helper executable is cached 
DEBU Running '/home/admin/.crc/bin/crc-admin-helper-linux --version' 
DEBU Found crc-admin-helper-linux version v0.0.7  
DEBU crc-admin-helper executable already cached   
INFO Checking for obsolete admin-helper executable 
DEBU Checking if an older admin-helper executable is installed 
DEBU No older admin-helper executable found       
INFO Checking if running on a supported CPU architecture 
INFO Checking minimum RAM requirements            
DEBU Total memory of system is 10074533888 bytes  
INFO Checking if crc executable symlink exists    
INFO Checking if Virtualization is enabled        
DEBU Checking if the vmx/svm flags are present in /proc/cpuinfo 
DEBU CPU virtualization flags are good            
INFO Checking if KVM is enabled                   
DEBU Checking if /dev/kvm exists                  
DEBU /dev/kvm was found                           
INFO Checking if libvirt is installed             
DEBU Checking if 'virsh' is available             
DEBU 'virsh' was found in /usr/bin/virsh          
DEBU Checking 'virsh capabilities' for libvirtd/qemu availability 
DEBU Running 'virsh --readonly --connect qemu:///system capabilities' 
DEBU Found x86_64 hypervisor with 'hvm' capabilities 
INFO Checking if user is part of libvirt group    
DEBU Checking if current user is part of the libvirt group 
DEBU Current user is already in the libvirt group 
INFO Checking if active user/process is currently part of the libvirt group 
INFO Checking if libvirt daemon is running        
DEBU Checking if libvirtd service is running      
DEBU Running 'systemctl status virtqemud.socket'  
DEBU Command failed: exit status 3                
DEBU stdout: * virtqemud.socket - Libvirt qemu local socket
   Loaded: loaded (/usr/lib/systemd/system/virtqemud.socket; disabled; vendor preset: disabled)
   Active: inactive (dead)
   Listen: /run/libvirt/virtqemud-sock (Stream) 
DEBU stderr:                                      
DEBU virtqemud.socket is neither running nor listening 
DEBU Running 'systemctl status libvirtd.socket'   
DEBU libvirtd.socket is running                   
INFO Checking if a supported libvirt version is installed 
DEBU Checking if libvirt version is >=3.4.0       
DEBU Running 'virsh -v'                           
INFO Checking if crc-driver-libvirt is installed  
DEBU Checking if crc-driver-libvirt is installed  
DEBU Running '/home/admin/.crc/bin/crc-driver-libvirt version' 
DEBU Found crc-driver-libvirt version 0.13.1      
DEBU crc-driver-libvirt is already installed      
INFO Checking crc daemon systemd service          
DEBU Checking crc daemon systemd service          
DEBU Checking if crc-daemon.service is running    
DEBU Running 'systemctl --user status crc-daemon.service' 
DEBU Command failed: exit status 3                
DEBU stdout: * crc-daemon.service - CodeReady Containers daemon
   Loaded: loaded (/home/admin/.config/systemd/user/crc-daemon.service; static; vendor preset: enabled)
   Active: inactive (dead) 
DEBU stderr:                                      
DEBU crc-daemon.service is neither running nor listening 
DEBU Checking if crc-daemon.service has the expected content 
INFO Checking crc daemon systemd socket units     
DEBU Checking crc daemon systemd socket units     
DEBU Checking if crc-http.socket is running       
DEBU Running 'systemctl --user status crc-http.socket' 
DEBU crc-http.socket is listening                 
DEBU Checking if crc-http.socket has the expected content 
DEBU Checking if crc-vsock.socket is running      
DEBU Running 'systemctl --user status crc-vsock.socket' 
DEBU crc-vsock.socket is listening                
DEBU Checking if crc-vsock.socket has the expected content 
INFO Checking if systemd-networkd is running      
DEBU Checking if systemd-networkd.service is running 
DEBU Running 'systemctl status systemd-networkd.service' 
DEBU Command failed: exit status 4                
DEBU stdout:                                      
DEBU stderr: Unit systemd-networkd.service could not be found. 
DEBU systemd-networkd.service is not running      
INFO Checking if NetworkManager is installed      
DEBU Checking if 'nmcli' is available             
DEBU 'nmcli' was found in /usr/bin/nmcli          
INFO Checking if NetworkManager service is running 
DEBU Checking if NetworkManager.service is running 
DEBU Running 'systemctl status NetworkManager.service' 
DEBU NetworkManager.service is already running    
INFO Checking if dnsmasq configurations file exist for NetworkManager 
INFO Checking if the systemd-resolved service is running 
DEBU Checking if systemd-resolved.service is running 
DEBU Running 'systemctl status systemd-resolved.service' 
DEBU systemd-resolved.service is already running  
INFO Checking if /etc/NetworkManager/dispatcher.d/99-crc.sh exists 
DEBU Checking NetworkManager dispatcher file for crc network 
DEBU Dispatcher file has the expected content     
INFO Checking if libvirt 'crc' network is available 
DEBU Checking if libvirt 'crc' network exists     
DEBU Running 'virsh --connect qemu:///system net-info crc' 
DEBU Checking if libvirt 'crc' definition is up to date 
DEBU Running 'virsh --connect qemu:///system net-dumpxml --inactive crc' 
DEBU libvirt 'crc' network has the expected value 
INFO Checking if libvirt 'crc' network is active  
DEBU Checking if libvirt 'crc' network is active  
DEBU Running 'virsh --connect qemu:///system net-info crc' 
DEBU libvirt 'crc' network is already active      
INFO Checking if CRC bundle is extracted in '$HOME/.crc' 
DEBU Development build, skipping check            
Your system is correctly setup for using CodeReady Containers, you can now run 'crc start -b $bundlename' to start the OpenShift cluster

```

Please consider posting the output of `crc start --log-level debug`  on http://gist.github.com/ and post the link in the issue.

**Comment by praveenkumar** [comment]
@tumimo-cloud Looks like you are not using the released version of crc which have bundle embedded in the binary itself.
```
CodeReady Containers version: 1.34.0+34c31851
OpenShift version: 4.9.0 (not embedded in executable)
```

**Comment by gbraad** [comment]
Also, note that nested virtualization is not actually supported.

**Comment by gbraad** [comment]
> Looks like you are not using the released version of crc 

Where did you get this release from?

**Comment by tumimo-cloud** [comment]
Hi Gerard, I got the copy from here (RH Cloud Console):https://developers.redhat.com/products/codeready-containers/overviewbut now I see there’s one over here:https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/crc/latest/Regards,TumeloM From: Gerard BraadSent: Monday, 15 November 2021 14:59To: code-ready/crcCc: tumimo-cloud; MentionSubject: Re: [code-ready/crc] CRC setup failing to deploy correctly [BUG] (Issue #2837) Looks like you are not using the released version of crcWhere did you get this release from?—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by tumimo-cloud** [comment]
Thanks Gerard. I noticed that when I set up in VMware. VMware documentation is very clear about that. From: Gerard BraadSent: Monday, 15 November 2021 14:48To: code-ready/crcCc: tumimo-cloud; MentionSubject: Re: [code-ready/crc] CRC setup failing to deploy correctly [BUG] (Issue #2837) Also, note that nested virtualization is not actually supported.—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by praveenkumar** [comment]
@tumimo-cloud Should we close this issue then?

**Comment by tumimo-cloud** [comment]
More issues...

Thanks @praveenkumar. I downloaded another copy. 
There is however another issue... seems to be DNS config.
The deployment seems to have succeed once 'crc start' runs. It however breaks once it reaches: 'INFO Starting OpenShift kubelet service' . See below debug log:

`[admin@localhost ~]$ crc status --log-level debug
DEBU CodeReady Containers version: 1.32.1+4d40571f 
DEBU OpenShift version: 4.8.5 (embedded in executable) 
DEBU Running 'crc status'                         
DEBU Checking file: /home/admin/.crc/machines/crc/.crc-exist 
DEBU Checking file: /home/admin/.crc/machines/crc/.crc-exist 
DEBU Found binary path at /home/admin/.crc/bin/crc-driver-libvirt 
DEBU Launching plugin server for driver libvirt   
DEBU Plugin server listening at address 127.0.0.1:35687 
DEBU () Calling .GetVersion                       
DEBU Using API Version 1                          
DEBU () Calling .SetConfigRaw                     
DEBU () Calling .GetMachineName                   
DEBU (crc) Calling .GetState                      
DEBU (crc) DBG | time="2021-11-24T21:39:40+02:00" level=debug msg="Getting current state..." 
DEBU (crc) DBG | time="2021-11-24T21:39:40+02:00" level=debug msg="Fetching VM..." 
DEBU (crc) Calling .GetBundleName                 
DEBU (crc) Calling .GetIP                         
DEBU (crc) DBG | time="2021-11-24T21:39:40+02:00" level=debug msg="GetIP called for crc" 
DEBU (crc) DBG | time="2021-11-24T21:39:40+02:00" level=debug msg="Getting current state..." 
DEBU (crc) DBG | time="2021-11-24T21:39:40+02:00" level=debug msg="IP address: 192.168.130.11" 
DEBU Running SSH command: df -B1 --output=size,used,target /sysroot | tail -1 
DEBU Using ssh private keys: [/home/admin/.crc/machines/crc/id_ecdsa /home/admin/.crc/cache/crc_libvirt_4.8.5/id_ecdsa_crc] 
DEBU SSH command results: err: <nil>, output: 32737570816 4944076800 /sysroot 
DEBU cannot get OpenShift status: stat /home/admin/.crc/machines/crc/kubeconfig: no such file or directory 
DEBU Making call to close driver server           
DEBU (crc) Calling .Close                         
DEBU Successfully made call to close driver server 
DEBU Making call to close connection to plugin binary 
CRC VM:          Running
OpenShift:       Unreachable (v4.8.5)
Disk Usage:      4.944GB of 32.74GB (Inside the CRC VM)
Cache Usage:     15.96GB
Cache Directory: /home/admin/.crc/cache
`

**Comment by praveenkumar** [comment]
@tumimo-cloud This is a different issue, I would suggest you to create a new issue with the template. Closing this one.
